PlaywrightEvershop/Tests/ProductCheckoutTest.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.Playwright;
+using PlaywrightEverShop.Models;
+using PlaywrightEverShop.PageObjects;
+
+namespace PlaywrightEverShop.Tests;
+
+internal class ProductCheckoutTest : UiTestFixture
+{
+    private EverShopMenPage _everShopMenPage;
+    private EverShopCartPage _everShopCartPage;
+
+    [SetUp]
+    public new void Setup()
+    {
+        _everShopMenPage = new EverShopMenPage(Page);
+        _everShopCartPage = new EverShopCartPage(Page);
+    }
+
+    [Test]
+    public async Task ProductCheckout()
+    {
+        // Arrange
+        await Page.APIRequest.PostAsync("https://demo.evershop.io/customer/login", new APIRequestContextOptions
+        {
+            DataObject = new UserModel
+            {
+                Email = "lichar@ukr.net",
+                Password = "Qwerty12345*"
+            }
+        });
+
+        await _everShopMenPage.GoToMenPage();
+        const int productIndex = 0;
+        
+        // Act
+        await _everShopMenPage.SelectProduct(productIndex);
+        var productNameFromPage = await _everShopMenPage.GetProductName();
+        await Page.WaitForTimeoutAsync(1500); // Cannot pick size without waiting
```

Comment:
even if you have to put hardcoded timeout, it should be inside SelectSizeOption
you can wait for element with class "selected" instead of hardcoded timeout
![image](https://github.com/user-attachments/assets/94bead54-b579-42a3-864a-22b7567272c0)